src/assets/img/skull.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
1,2,3 lines can be removed for a little optimalization (not full, but for now it will be enough).